packages/components/src/spectrum/comboBox/ComboBox.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  ComboBox as SpectrumComboBox,
+  SpectrumComboBoxProps,
+} from '@adobe/react-spectrum';
+import type { FocusableRef } from '@react-types/shared';
+import cl from 'classnames';
+import type { NormalizedItem } from '../utils';
+import { PickerPropsT, usePickerProps } from '../picker';
+
+export type ComboBoxProps = PickerPropsT<SpectrumComboBoxProps<NormalizedItem>>;
```

Comment:
I think we still want to template the props here so items get passed through correctly no?
Something like:
```suggestion
export type ComboBoxProps<T extends NormalizedItem = NormalizedItem> = PickerPropsT<SpectrumComboBoxProps<T>>;
```

Reply:
What's the use case? `SpectrumComboBoxProps<NormalizedItem>` should already pass things through. We aren't extending `NormalizedItem` so I think it should be ok?

Reply:
I guess you're right - I was thinking if you pass in items that extended items and wanted the original item back, but that's not how this works anyway.